plc4j/core/src/main/java/org/apache/plc4x/java/PlcDriverManagerInterface.java

```diff
@@ -0,0 +1,29 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+package org.apache.plc4x.java;
+
+import org.apache.plc4x.java.api.authentication.PlcAuthentication;
+import org.apache.plc4x.java.api.connection.PlcConnection;
+import org.apache.plc4x.java.api.exceptions.PlcConnectionException;
+
+public interface PlcDriverManagerInterface {
```

Comment:
I dont like the name that much, but I see your problem (as we want to keep the name of the "original" PlcDriverMangager). Perhaps it is best to simply subclass the existing PlcDriverManager as we want to allow users to use the Proxy AS PlcDriverManager. So it should be:
`PlcDriverManager manager = new ...`